yotemplate/yotemplate_test.go

```diff
@@ -0,0 +1,57 @@
+package yotemplate_test
+
+import (
+	"github.com/juandunbar/yobirthday/yodata"
+	"github.com/juandunbar/yobirthday/yotemplate"
+	"testing"
+)
+
+const (
+	defaultEmail = "\nHey Tyler,\nHope this email finds you well. Just wanted to say Happy Birthday!\n\n- David Singleton\n"
+	familyEmail = "\nHey Mom,\nWow another year in the books.  Ill have to come visit soon.\n\nLove you, David\n"
+	friendEmail = "\nYo chaddy,\nWhats up?  Happy Birthday!! Let's grab a beer soon, ttyl.\n\n- dmoney\n"
```

Comment:
If another email template was added that did not contain newlines it would have to reflect that in the test as its testing the exact output from the render.  I could trim those characters before comparison to allow more freedom, but i generally like to err on the side of caution.